packages/core/src/Game/World.js

```diff
@@ -0,0 +1,429 @@
+const THREE = require('three');
+const WorldState = require('./WorldState');
+const { Collisions } = require('detect-collisions');
+const GameObject = require('./GameObject/GameObject').GameObject;
+const Collider = require('./GameObject/Components/Collider');
+const WorldScript = require('./GameObject/Components/WorldScript');
+
+// jsdoc import
+const WorldContext = require('./WorldContext');
+
+/**
+ * @typedef {object} WorldOrigin
+ * @property {number} lng - longitude coordinate of the origin.
+ * @property {number} lat - latitude coordinate of the origin.
+ * @property {number} alt - altitude coordinate of the origin.
+ */
+
+/**
+ * Represent a world model and methods to simulated it given a {@link WorldContext}
+ */
+const World = class {
+  /**
+   *
+   * @param {object|JSON} json - The json object that contains the world's data.
+   * @param {object|JSON} json.gameObject - JSON to create the {@link GameObject} of the world.
+   * @param {string} json.name - Name of the world. (optional)
+   * @param {string} json.uuid - UUID of the world. (optional)
+   * @param {WorldOrigin} json.origin - Origin of the world. (optional)
```

Comment:
precise you need pass it in order to initialize the itownsView ? (we are in core there are no notion of itowns here)

Reply:
origin should then maybe be remove and be declare in config.gameView ?
maybe take the extent and place the world ref at the center ?
world should ref their config path in order to point their geo loc ?